docassemble/MotionToStayEviction/data/questions/review_screen.yml

```diff
@@ -0,0 +1,386 @@
+---
+event: review_case_info
+id: |-
+  review screen
+question: |-
+  Review your answers
+review:
+  - Edit: users.revisit
+    button: |
+      **Your party: (Edit to change name and address)**
+
+      %  for my_var in users:
+        * ${ my_var.name }
+      % endfor
+  - Edit: other_parties.revisit
+    button: |
+      **The other party: (Edit to change name and address)**
+      
+      % for my_var in other_parties:
+        * ${ my_var.name }
+      % endfor
+  - Edit: users[0].address.address
+    button: |
+      **Your address:**
+      ${ users[0].address.on_on_line() }
+  - Edit: |-
+      case_new_or_existing
+    button: |
+      **Is this a new case?**
+
+      ${ showifdef('case_new_or_existing') }
+  - Edit: |-
+      users.there_is_another
```

Comment:
Not sure we want this to be editable, it should either be in the user table, or just not present (since we don't want people to add parties to the case accidentally).